package.json

```diff
@@ -29,24 +43,33 @@
     "*.js": "eslint --cache --fix",
     "*.{js,json,md,yml}": "prettier --write"
   },
-  "eslintConfig": {
+  "npmpackagejsonlint": {
     "extends": [
-      "stylelint"
+      "@stylelint/npm-package-json-lint-config"
     ]
   },
-  "dependencies": {
```

Comment:
I think we should continue having Prettier automatically format our package.json, including the sorting of it, and remove any formatting/sorting related rules from the shared npm-package-json-lint config.

Reply:
Ugh, this was meant to remain, I removed it in my first attempted PR, will restore as the two of these together work brilliantly :)

Reply:
Restored in 0183545